bundles/org.openhab.transform.basicprofiles/README.md

```diff
@@ -242,6 +242,9 @@ The `LHS_OPERAND` and the `RHS_OPERAND` can be either one of these:
   This can be customized by specifying the "window size" or sample count applicable to the function, e.g. `$MEDIAN(10)` will return the median of the last 10 values.
   All the functions except `$DELTA` support a custom window size.
 
+In the case of comparisons and calculations involving `QuantityType` and `DecimalType` values, both operand and incoming value must be of the same type.
```

Comment:
It seems that this paragraph can / should be merged with the next one simply to avoid being repetitive.

```suggestion
In the case of comparisons and calculations involving `QuantityType` values, both operands, whether they are Item states, the incoming value, or constants, must be of the same type and have compatible units.
```